editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper-third-party.spec.browser2.tsx

```diff
@@ -37,6 +21,17 @@ import {
 import { addFileToProjectContents } from '../../assets'
 import { StoryboardFilePath } from '../../editor/store/editor-state'
 import { applyUIDMonkeyPatch } from '../../../utils/canvas-react-utils'
+import { BuiltInDependencies } from '../../../core/es-modules/package-manager/built-in-dependencies-list'
+import { matchInlineSnapshotBrowser } from '../../../../test/karma-snapshots'
+
+BuiltInDependencies.push({
```

Comment:
this replaces the old jest.mock . while we are looking for a working module mocking solution, I realized that in this particular case we did not need one.